src/core/services/user.service.ts

```diff
@@ -0,0 +1,57 @@
+import router from "@/router"
+import axios from "axios"
+import { environment } from "@/environments/environment"
+
+
+export function loginUser(emailInput: string, passwordInput: string) {
+    const loginData = {
+        "email": emailInput,
+        "password": passwordInput
+    }
+
+    axios({
+        method: "POST",
+        url: environment.baseUrl + "/login",
+        data: loginData
+    })
+        .then(function (response) {
+            console.log(response.data.isSuccessful)
+            if (response.data.isSuccessful) {
+                router.push("/home")
+            }
+            else if (!response.data.isSuccessful) {
+                router.push("/login")
+            }
```

Comment:
Please trnasfer the page naviagtion in the page. All logic should just be the token management and pre-login stuffs.

Reply:
Done.